x-pack/plugins/apm/kibana.jsonc

```diff
@@ -46,7 +46,8 @@
       "taskManager",
       "usageCollection",
       "customIntegrations", // Move this to requiredPlugins after completely migrating from the Tutorials Home App
-      "licenseManagement"
+      "licenseManagement",
+      "profiling"
```

Comment:
Generally, I think this is an unsustainable and unscalable pattern for us to be doubling down on... am I right in my understanding that this dependency introduction is solely for access to the exported locators?